beaver/transports/stomp_transport.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+import stomp
+
+from beaver.transports.base_transport import BaseTransport
+from beaver.transports.exception import TransportException
+
+
+class StompTransport(BaseTransport):
+
+    def __init__(self, beaver_config, logger=None):
+        """
+        Mosquitto client initilization. Once this this transport is initialized
+        it has invoked a connection to the server
+        """
+        super(StompTransport, self).__init__(beaver_config, logger=logger)
+
+        self.createConnection(beaver_config)
+        self.logger = logger
+
+ 
+    def callback(self, filename, lines, **kwargs):
+        """publishes lines one by one to the given topic"""
+        timestamp = self.get_timestamp(**kwargs)
+        if kwargs.get('timestamp', False):
+            del kwargs['timestamp']
+        
+
+        for line in lines:
+            try:
+                import warnings
+                with warnings.catch_warnings():
+                    warnings.simplefilter('error')
+                    m = self.format(filename, line, timestamp, **kwargs)
+                    self.logger.debug("Sending message " + m)
+                    self.conn.send(destination=self.queue, body=m)    
```

Comment:
The redis transport implements it, which is good as it reduces network chatter - as you mentioned - and allows us to get more throughput. If it's not possible with stomp, then it's not possible :(